src/main/java/uk/gov/companieshouse/service/dissolution/chips/ChipsResponseService.java

```diff
@@ -48,7 +49,13 @@ public void saveAndNotifyDissolutionApplicationOutcome(ChipsResponseCreateReques
         dissolution.setActive(false);
 
         if (featureToggleConfig.isRefundsEnabled() && dissolutionVerdict.getResult() == VerdictResult.REJECTED) {
-            dissolutionRefundService.handleRefund(dissolution);
+            try {
+                dissolutionRefundService.handleRefund(dissolution);
+            } catch (WebClientException e) {
+                logger.info(String.format("Automatic refund failed, sending rejection email to finance. Exception: %s", e.getMessage()));
+
+                dissolutionEmailService.sendRejectionEmailToFinance(dissolution, dissolutionVerdict);
```

Comment:
not a big deal, but is there anyway we can potentially reuse the same email object here and on L65?

Reply:
Updated. Is this what you were thinking about?

Reply:
Nope - underneath the hood this calls `getApplicationRejectedEmailDocument`. At the moment we are currently executing this twice and getting to same value twice. What I was getting at was something like:
```
final EmailDocument rejectedEmail = // get email data object

try {
  // refund
} catch () {
  rejectedEmail.send(finance)
}

rejectedEmail.send(applicant)
```
therefore we would create the object once and reuse it.

Although this may expose the lower-workings of the email functionality outside of the DissolutionEmailService. This may prove more messy. I'd revert this back to what you had before as it was cleaner.